drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_dma.c

```diff
@@ -284,9 +279,6 @@ static int transceive_dma_sync(const struct device *dev)
 	}
 
 	spi_context_cs_control(ctx, false);
-
-	base->TCR &= ~LPSPI_TCR_CONT_MASK;
-
```

Comment:
I'm pretty sure this should stay where it is for the synchronous transfer, this is the point where the transfer is over, not during the while loop / dma callbacks.

Reply:
The point is: if put it some other place, loopback test fails on RT platform. Please let me know if you find a better idea and verified it.  This modification works fine on both MCXN and RT11xx.